charts/consul/values.yaml

```
@@ -596,6 +596,19 @@ server:
   # @type: array<map>
   extraVolumes: []
 
+  # A list of sidecar containers.
+  # @type: array<string>
```

Since this is an array of objects/maps
```suggestion
  # @type: array<map>
```

---

I think the `@type` line needs to be the last in the comment. For example:

```yaml
  # Example:
  #
  # ```yaml
  # imagePullSecrets:
  #   - name: pull-secret-name
  #   - name: pull-secret-name-2
  # ```
  # @type: array<map>
```